### DISCUSSION TITLE
There’s a Type-C slot mega 2560 pro in AliExpress now

### DISCUSSION THREAD

**Discussion by solleo1989** [sanni/cartreader#860] [Hardware]
I've noticed there's a type-C version Mega 2560 pro in AliExpress month back. 
![image](https://github.com/sanni/cartreader/assets/5420867/0bc00be9-b967-40b9-9d20-102f38922848)

I bought one, desoldered the fuse and soldered back a Dupont wire. It works as same as the micro-usb version, but more user friendly(I don't like microusb)

Just a heads up for anyone wanna a type-c version HW5.
![image](https://github.com/sanni/cartreader/assets/5420867/b523f26c-2713-44a6-8fc0-0985648792e2)
![image](https://github.com/sanni/cartreader/assets/5420867/7a6fc42b-80c4-4907-94be-9f89ff8fb3fb)
![image](https://github.com/sanni/cartreader/assets/5420867/b773753c-c697-4d25-a2c8-c4105182e5bb)

**Comment by mgiambrone** [comment]
![20231213_101918](https://github.com/sanni/cartreader/assets/11039710/007a4ae3-2554-4b1d-acfc-6ee9c588dda2)

![17024814012644151281017188260061](https://github.com/sanni/cartreader/assets/11039710/94062a09-297a-499a-bfe8-fa9f82ffa1a7)

I received this one from Bonzo's retro shop, called "MEGA 2560 PRO (EM?ED)" Seems that in the version I have, one of the components is rotated 90 degrees (it's next to the underlined 4 SMD fuse  that needs to be removed)

**Reply by Ancyker** [reply]
It says Embed, btw.

**Reply by mgiambrone** [reply]
> It says Embed, btw.

That's a b? Looks like a reverse R with the top filled in

**Reply by Ancyker** [reply]
Yep.

![image](https://github.com/sanni/cartreader/assets/7105333/0bc2864d-9666-4ee7-ab29-df2ab64b97ea)
